commit c0c0cb2fd6d9e228267b2762916f54e84b1fd037
Author: Olly Betts <olly@survex.com>
Date:   Wed May 18 11:10:00 2022 +1200

    Fix red line in clino background to be grey
    
    The red line was an accidentally committed change made while debugging a
    drawing issue.

diff --git a/src/gfxcore.cc b/src/gfxcore.cc
index ba340eb9..b785713d 100644
--- a/src/gfxcore.cc
+++ b/src/gfxcore.cc
@@ -751,11 +751,10 @@ void GfxCore::DrawClinoBack() {
 	DrawTick(angle);
     }
 
-    SetColour(col_RED);
+    SetColour(col_GREY);
     glaCoord r = INDICATOR_RADIUS * GetContentScaleFactor();
     PlaceIndicatorVertex(0, r);
     PlaceIndicatorVertex(0, -r);
-    SetColour(col_GREY);
     PlaceIndicatorVertex(0, 0);
     PlaceIndicatorVertex(r, 0);
 
